### PR TITLE
Update example scripts

### PR DESCRIPTION
**Pull request by Foxglove144**
# Description

1. In adversarial_training_FBF.py, added correct import statement for PyTorchClassifier.
2. In adversarial_training_data_augmentation.py, disabled TensorFlow eager execution. 
3. In get_started_xgboost.py, added correct eval_metric paramters.
4. In mnist_cnn_fgsm.py, disable TensorFlow eager execution and demonstrate FGSM over multiple epsilon values.
5. In mnist_poison_detection.py, disable TensorFlow eager execution.
6. In mnist_transferability.py, disable TensorFlow eager execution.
7. In get_started_lightgbm.py, set force_row_wise=true in params to remove the overhead

Fixes # (issue)

## Type of change

Please check all relevant options.

- [ ] Improvement (non-breaking)
- [x] Bug fix (non-breaking)
- [ ] New feature (non-breaking)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Testing

Please describe the tests that you ran to verify your changes. Consider listing any relevant details of your test configuration.

- [ ] Test A
- [ ] Test B

**Test Configuration**:
- OS
- Python version
- ART version or commit number
- TensorFlow / Keras / PyTorch / MXNet version

# Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [x] I have commented my code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: examples/mnist_cnn_fgsm.py [hunk 1]
```diff
@@ -35,12 +39,16 @@
...
 
-# Evaluate the classifier on the adversarial examples
-preds = np.argmax(classifier.predict(x_test_adv), axis=1)
-acc = np.sum(preds == np.argmax(y_test, axis=1)) / y_test.shape[0]
-print("\nTest accuracy on adversarial sample: %.2f%%" % (acc * 100))
+# Iterate over epsilon values
+for epsilon in epsilon_values:
+    # Craft adversarial samples with FGSM
+    adv_crafter = FastGradientMethod(classifier, eps=epsilon)
+    x_test_adv = adv_crafter.generate(x=x_test)
```
Comment (beat-buesser) [inline examples/mnist_cnn_fgsm.py:49]: we could make the attack even stronger by attacking the true labels
```suggestion
    x_test_adv = adv_crafter.generate(x=x_test, y=y_test)
```

**Review by beat-buesser — CHANGES_REQUESTED** [review]
Hi @Foxglove144 Thank you very much for updating all example scripts. I like the extension to the FGSM example on MNIST where I have added a small suggestion to make the attack stronger. What do you think?

**Comment by Foxglove144** [comment]
i'm sorry I seem to have accidentally committed your changes without signing off and I messed up trying to revert the file. Can you please tell me how to fix this?

**Comment by beat-buesser** [comment]
Hi @Foxglove144 I think something went wrong with the last signing-off of commits that signed the commits of dependabot. Could you please move your commits with`git cherry-pick` to a new branch so that I can merge your changes without the double-signed commits?
